qe_metrics/utils/issue_utils.py

```diff
@@ -81,6 +82,8 @@ def create_update_issues(
     """
     Create or update JiraIssuesEntity items in the database from a list of Jira issues. Sets status of obselete issues as "obselete".
 
+    Only creates issues if they are of type "bug" and if they are less than DATA_RETENTION_DAYS_OLD days old.
```

Comment:
looking at `of type "bug"` - the user provides us with the query, a team can decide if the type is `bug` or anything else. so why do we restrict to this type?

Reply:
I think we have gone back and forth on whether to allow non-bugs into the database in our weekly sync meetings. We may just want to bring this up in the meeting tomorrow and get a definitive answer from the group. [Here is some context from the last PR](https://github.com/RedHatQE/qe-metrics/pull/59#discussion_r1543560218).

Reply:
Done in #78